Tasks/DependencyCheck/dependency-check-build-task.ts

```diff
@@ -127,10 +127,7 @@ async function run() {
             hasLocalInstallation = false;
             localInstallPath = tl.resolve('./dependency-check');
 
-            if (!tl.exist(localInstallPath)) {
-                console.log(`Creating dependency check installer directory at ${localInstallPath}`);
-                tl.mkdirP(localInstallPath);
-            }
+            tl.checkPath(localInstallPath, 'Dependency Check installer');
```

Comment:
Change seems to brake our pipeline. Missing install path leads to an unhandled exception. ``checkPath`` does not create a directory, only throws an exception if not available

![grafik](https://github.com/dependency-check/azuredevops/assets/166133745/8eeb92e9-d0ac-48cc-ba2f-3b1ae7f51b97)